pr_etiquette.md

```diff
@@ -0,0 +1,63 @@
+# Etiquette for pull requests
+
+## As an author
+
+Submitting pull requests in EDC should be done while adhering to a couple of simple rules.
+
+- Familiarize yourself
+  with [coding style](styleguide.md), [architectural patterns](docs/development/coding-principles.md) and
+  other contribution guidelines.
+- No surprise PRs please. Before you submit a PR, open a discussion or an issue outlining your planned work and give
+  people time to comment. It may even be advisable to contact committers using the `@mention` feature. Unsolicited PRs
+  may get ignored or rejected.
+- Create your working branch in your fork of TractusX-EDC, and create the PR against the upstream `develop` branch
+- Create focused PRs: your work should be focused on one particular feature or bug. Do not create broad-scoped PRs that
+  solve multiple issues as reviewers may reject those PR bombs outright.
+- Provide a clear description and motivation in the PR description in GitHub. This makes the reviewer's life much
+  easier. It is also helpful to outline the broad changes that were made, e.g. "Changes the schema of XYZ-Entity:
+  the `age` field changed from `long` to `String`".
+- If you introduce new 3rd party dependencies, be sure to note them in the PR description and explain why they are
+  necessary.
```

Comment:
Regarding https://eclipse-tractusx.github.io/docs/oss/code-reviews#step-1---content-review-the-responsibility-lies-with-the-product-team (fourth point) does we need to add this here as well?

Reply:
Those rules from the link seem very coarse to me. While having strict rules can seem "gate-keepy", experience in EDC has shown that having strict rules of contribution is a good thing, especially regarding PRs. 

Loose rules can lead to wasted (committer) time, pointless discussions and frustration.

Reply:
The rules @florianrusch-zf are actually also up for dicussion the repository behind the website is [eclipse-tractusx/eclipse-tractusx.github.io](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/). 
Suggestions highly welcome. I think having the same set of rules for the whole eclipse-tractusx project instead of distinct sets per repo will make it easier for contributors.